src/dials/algorithms/scaling/scale_and_filter.py

```diff
@@ -498,6 +498,8 @@ def _add_new_histogram(d, hist, index):
         )
 
     for c, deltas in enumerate(delta_cc_half_lists):
+        if not (max(deltas) > min(deltas)):
```

Comment:
Is this not just a slightly roundabout way of checking that all elements are equal? If you're going to make a `flex.double` of the `deltas` anyway, you could do `deltas.all_eq(deltas[0])`?

Reply:
Good point, i'll update the code